index.html

```diff
@@ -594,52 +596,57 @@
 \end{equation}
-The physical RGB scattering coefficient is then given by $\boldsymbol{\mu}_s = \boldsymbol{\alpha}\,\boldsymbol{\mu}_t$.
+Thus since $0 \le \mathbf{C} \le 1$, $\mathbf{E}_\mathrm{subsurface} \le 1$ so energy is always conserved, and if $\mathbf{C}=1$ then $\mathbf{E}_\mathrm{subsurface}=1$ which guarantees that a white **`subsurface_color`** passes the furnace test. According to this definition, the **`subsurface_color`** $\mathbf{C}$ is the observed reflection color (viewed at normal incidence under uniform illumination) in areas where the Fresnel reflection is negligible, and otherwise the observed color is a blend of $\mathbf{C}$ with the gray Fresnel reflection that conserves total reflected energy.
 
-This relationship was derived (via fitting) in the isotropic $g=0$ case. To account for anisotropy, similarity theory [#d'Eon2021] shows that a medium with anisotropy $g$ will have approximately similar overall light transport to an isotropic medium (while still exhibiting noticeably different directional effects, especially in low-order scattering) provided that the scattering coefficient is remapped to $\bar{\boldsymbol{\mu}}_s = \boldsymbol{\mu}_s / (1 - g)$, while keeping the absorption coefficient the same. Applying this, the net result is that the remapped single-scattering albedo $\bar{\boldsymbol{\alpha}}$ is given by
+Given the required multiple-scattering albedo $\mathbf{E}_\mathrm{multi-scatter}$ according to the formula above, then in principle, the single-scattering albedo $\boldsymbol{\alpha}$ which generates the required $\mathbf{E}_\mathrm{multi-scatter}$ can be determined. We do not require any particular theoretical formula for $\boldsymbol{\alpha}(\mathbf{C})$ be used, but the closer it is to satisfying equation [subsurface_albedo_constraint] the closer it is to the ground truth. Ideally, this formula should take into account all the properties of the medium as well as the dielectric interface. A number of such approximate formulas have been derived in the literature. A well-known approximation is due to van de Hulst ([#Kulla2017], [#d'Eon2021]), which assumes an index-matched boundary (i.e. $\mathbf{E}_\mathrm{spec} = 0$, thus $\mathbf{E}_\mathrm{multi-scatter} = \mathbf{C}$), where:
 \begin{equation}
-\bar{\boldsymbol{\alpha}} = \frac{\boldsymbol{\alpha}} {1  - g (1 - \boldsymbol{\alpha})} \ ,
+\mathbf{C} = \frac{(1 - \mathbf{s})(1 - 0.139 \mathbf{s})}{1 + 1.17 \mathbf{s}} \ , \quad
+\mathbf{s} = \sqrt{\frac{1 - \boldsymbol{\alpha}}{1 - \boldsymbol{\alpha}\, g}} \ .
 \end{equation}
-and remapped extinction coefficient $\bar{\boldsymbol{\mu}}_t$ by
+Inversion of this gives the per-channel single-scatter albedo $\boldsymbol{\alpha}$ in terms of the **`subsurface_color`** $\mathbf{C}$ and anisotropy $g$:
 \begin{equation}
-\bar{\boldsymbol{\mu}}_t = \frac{1  - g (1 - \boldsymbol{\alpha})}{1 - g} \boldsymbol{\mu}_t \ .
+\boldsymbol{\alpha} = \frac{1 - \mathbf{s}^2}{1 - g\,\mathbf{s}^2} \ , \quad
```

Comment:
If I remember correctly, the inclusion of *g* implies in the formula implies that the albedo remapping should compensate for the anisotropy so that the observed color is equal to the specified **`subsurface_color`** regardless of the value of **`subsurface_anisotropy`**. However the extinction should not be affected by the **`subsurface_anisotropy`** (making the volume appear more translucent when it's more forward-scattering).

Is that correct? And do you think it's clear enough or would it be useful to state it more directly in the spec?

Reply:
Yes $g$ should be taken into account in the albedo remapping, since we said:

>Given the required multiple-scattering albedo ... according to the formula above, then in principle, the single-scattering albedo ... which generates the required [multiple-scattering albedo] can be determined...  Ideally, this formula should take into account all the properties of the medium as well as the dielectric interface.

So whatever the anisotropy is, the observed color should match the input. We did say fairly explicitly that the observed color is only affecting the single scattering albedo, but I think it could be further clarified if in this sentence

>Given the required multiple-scattering albedo $\mathbf{E}_\mathrm{multi-scatter}$   according to the formula above, 

>then in principle, the single-scattering albedo $\boldsymbol{\alpha}$ which generates the required $\mathbf{E}_\mathrm{multi-scatter}$ can be determined 

we add the the clarification:

>(assuming the given extinction $\boldsymbol{\mu}_t$ and anisotropy $g$).

I pushed that in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/44cb7a2a88cbb634c745d100c47720555d76b832.